test/e2e/pages/allocation-view.js

```diff
@@ -0,0 +1,29 @@
+import { Selector } from 'testcafe'
+
+import Page from './page'
+
+class AllocationViewPage extends Page {
+  getValueElementByKeyText(text) {
```

Comment:
This looks very similar to `page.getDlDefinitionByKey`. Can the existing method not be used to determine the same outcome?

Reply:
I hadn't seen that one. Will amend.